README.md

```diff
@@ -316,6 +318,45 @@ trait HasTranslations
 }
 ```
 
+### Fallback callback
+
+You can setup a fallback callback that is called when a translation key is missing/not found.
+Setting such a callback doesn't affect fallback behaviour and doesn't need fallback to be enabled.
+It just lets you execute some custom code like logging something or contact a remote service for example.
+
+First, you need to set it up in `config/translatable.php`:
+
+```
+return [
+    'fallback_callback_enabled' => true,
+];
+```
+
+Then you have to register some code you want to run, by passing a closure to `Spatie\Translatable\Facades\Translatable::fallback()`.
+
+Here's an example where we're going to register a closure that log a warning with some info about the
+missing translation key. Typically, you would put this in [a service provider of your own](https://laravel.com/docs/8.x/providers#writing-service-providers).
+
+```
+    // typically, in a service provider
+    
+    use Spatie\Translatable\Facades\Translatable;
+    
+    Translatable::fallback(function ($model, string $translationKey, string $locale) {
+    
+        // do something (ex: logging)
+        
+        Log::warning('Some translation key is missing from an eloquent model', [
+           'key' => $translationKey,
+           'locale' => $locale,
+           'model_id' => $model->id,
+           'model_class' => get_class($model), 
+        ]);
```

Comment:
I think it could be nice that if this closure returns something, that we use that as the translation.

Reply:
What do you mean ? Can you elaborate a bit ? thanx

Reply:
I mean that if we return a string in that closure, than that string should be used as the translation.

Reply:
I like that idea a lot. I wrote a dummy example but not sure if it what you had in mind ?

```
    // typically, in a service provider
    
    use Spatie\Translatable\Facades\Translatable;
    use App\Service\MyRemoteTranslationService;
    
    Translatable::fallback(function ($model, string $translationKey, string $locale, string $fallbackTranslation, string $fallbackLocale) {
        
        return MyRemoteTranslationService::getAutomaticTranslation($fallbackTranslation, $fallbackLocale, $locale);
    });
```

Reply:
maybe arguments should be ordered differently. Here is the API of a popular PHP package (https://github.com/stichoza/google-translate-php):

`GoogleTranslate::trans('Hello again', 'ka', 'en');`

Reply:
Order it in the way you think is best, and I'll comment.

I'm thinking that since `Translatable::fallback` is such a nice way handling non-translated stuff, that we could release a major version where we add that, and remove all other ways of defining fallbacks, that would make the package a bit simpler again.

Reply:
Do you think about removing config file completely ?

Reply:
Yeah 🙂 You feel up making the necessary changes?

Reply:
I'm up ! Using named parameter it could look something like:

```
Translatable::fallback(
    fallbackLocale: 'en',
    fallbackAny: true,
    missingKeyCallback: function ($model, string $translationKey, string $locale, string $fallbackTranslation, string $fallbackLocale) {
        
        return '?';
    },
);
```

Does it look right ?